src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -462,7 +462,19 @@ private void PackageOnExecuted(PackageManagerSearchElement element, PackageVersi
                     return;
                 }
 
-                var allPackageVersions = PackageManagerSearchElement.ListRequiredPackageVersions(headers, version);
+                List<Tuple<PackageHeader, PackageVersion>> allPackageVersions = null;
+                try
+                {
+                    allPackageVersions = PackageManagerSearchElement.ListRequiredPackageVersions(headers, version).ToList();
+                }
+                catch
+                {
+                    MessageBox.Show(String.Format("Could not find all dependencies for package {0}", element.Name),
```

Comment:
Hard coded, user facing message probably not a good idea..

Reply:
yes, please place into a resx so it can be localized.

Reply:
Also for the messaging, can we be more specific like `Could not find all dependencies for package based on versions specified by author`, just my two cents to avoid confusion